setup.py

```diff
@@ -21,20 +21,28 @@
         "tensorflow==2.9.0",
     ],
     "logging": [],
-    "model": ["numpy"],
-    "plot": ["pandas", "plotly"],
-    "processing": ["numpy", "pandas"],
+    "model": ["numpy==1.23.4"],
```

Comment:
shouldn't the requirements be `>=version`
I think having them == can lead to some problems if two library use two versions of the same dependency.

Reply:
To change it to `>=` I prefer to have it completly free